src/browzine-summon-adapter.js

```diff
@@ -98,56 +124,103 @@ angular.module("summonApp.directives")
     return coverImageUrl;
   };
 
-  function buildTemplate(data, browzineWebLink, bookIcon) {
-    var assetClass = "";
+  function buildTemplate(data, browzineWebLink) {
+    var wording = "";
+    var browzineWebLinkText = "";
+    var bookIcon = "https://s3.amazonaws.com/thirdiron-assets/images/integrations/browzine_open_book_icon.png";
 
-    // Customize the naming conventions for each type of item - Journal/Article - by changing the wording in the quotes below:
-    // E.g. You can customize "View the Journal" and "View Complete Issue".
     if(isJournal(data)) {
-      assetClass = "View the Journal";
+      wording = browzine.journalWording || "View the Journal";
+      browzineWebLinkText = browzine.journalBrowZineWebLinkText || "Browse Now";
     }
 
     if(isArticle(data)) {
-      assetClass = "View Complete Issue";
+      wording = browzine.articleWording || "View Complete Issue";
+      browzineWebLinkText = browzine.articleBrowZineWebLinkText || "Browse Now";
```

Comment:
If this statement is executed regardless of whether the object is a journal or article, we may want to take it out of the block.

Reply:
Should be separate customization properties `browzine.articleBrowZineWebLinkText` and `browzine.journalBrowZineWebLinkText`. Figured there could be a library that may want a different message for each scenario.